builtin/credential/ldap/path_login.go

```diff
@@ -0,0 +1,87 @@
+package ldap
+
+import (
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathLogin(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: `login`,
```

Comment:
I think we should do the same thing the `userpass` backend does, and put the username in the path. This allows us to revoke at a per-user level as well with the path based revoke.

https://github.com/hashicorp/vault/blob/master/builtin/credential/userpass/path_login.go#L13